x-pack/test_serverless/api_integration/test_suites/common/platform_security/views.ts

```diff
@@ -6,106 +6,128 @@
  */
 
 import expect from 'expect';
+import { InternalRequestHeader, RoleCredentials } from '../../../../shared/services';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getService }: FtrProviderContext) {
   const svlCommonApi = getService('svlCommonApi');
-  const supertest = getService('supertest');
+  const svlUserManager = getService('svlUserManager');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  let roleAuthc: RoleCredentials;
+  let internalReqHeader: InternalRequestHeader;
 
   describe('security/views', function () {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     describe('route access', () => {
       describe('disabled', () => {
```

Comment:
Same here: I don't have strong opinion, but having it recorded as skipped will provide some stats.

Reply:
https://github.com/elastic/kibana/pull/184935/commits/4290c99918aba0f46b15e14465f9f3b2347613b0